packages-exp/messaging-exp/package.json

```diff
@@ -16,13 +16,14 @@
     "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "build": "rollup -c && yarn api-report",
     "build:deps": "lerna run --scope @firebase/'{app-exp,messaging-exp}' --include-dependencies build",
+    "build:release": "rollup -c rollup.config.release.js && yarn api-report",
     "dev": "rollup -c -w",
     "test": "run-p test:karma type-check lint ",
     "test:integration": "run-p test:karma type-check lint && cd ../../integration/messaging && npm run-script test",
     "test:ci": "node ../../scripts/run_tests_in_ci.js",
     "test:karma": "karma start --single-run",
     "test:debug": "karma start --browsers=Chrome --auto-watch",
-    "prepare": "yarn build",
+    "prepare": "yarn build:release",
```

Comment:
Does this conflict with removing all prepares in the other PR?

Reply:
yeah, I will resolve the conflict in the other PR. It is an appropriate fix in the context of this PR.